README.md

```diff
@@ -101,6 +101,17 @@ Here in the Quick Start, we give a brief introduction of using source code to in
 
 ## Build from source
 
+Skip this paragraph if you are using Windows. As we use Thrift for our RPC module (communication and
+protocol definition), we involve Thrift during the compilation, so Thrift compiler 0.13.0 (or
+higher) is required to generate Thrift Java code. Thrift officially provides binary compiler for
+Windows, but unfortunately, they do not provide that for Unix OSs. However, we compiled a Unix
+compiler ourselves and put it onto GitHub, and with the help of a maven plugin, it will be
+downloaded automatically during compilation. This compiler works fine with gcc8 or later, Ubuntu
+MacOS, and CentOS, but previous versions and other OSs are not guaranteed. Should you find your gcc
+version or OS does not support the precompiled compiler, please upgrade your gcc version or follow the
+Thrift official instructions to compile the compiler yourself and rename it into `{project_root}\thrift\target\tools\thrift_0.12.0_0.13.0_linux.exe`.
```

Comment:
{project_root}\thrift\target\tools\thrift_0.12.0_0.13.0_linux.exe -> ``{project_root}\thrift\target\tools` ?

Reply:
Unfortuantely, the filename must be consistent, otherwise the plugin will not recognize it.